.gitignore

```
@@ -0,0 +1,3 @@
+*.lock
+
+charts/**/chart/charts/**.tgz
```

Could we just do this?
```suggestion
*.tgz
```

---

We could yes.